src/lib/phdi-building-blocks/tests/test_linkage.py

```diff
@@ -1,4 +1,7 @@
-from phdi_building_blocks.linkage import generate_hash_str, add_patient_identifier
```

Comment:
Are we missing a test for `get_field_with_use`?

Reply:
See prior comments regarding our handling of necessary tests.